rules/windows/file/file_event/file_event_win_creation_deno.yml

```
@@ -0,0 +1,30 @@
+title: Suspicious Deno File Written from Remote Source
+id: 6c0ce3b6-85e2-49d4-9c3f-6e008ce9796e
+status: experimental
+description: |
+    Detects Deno writing a file from a direct HTTP(s) call and writing to the appdata folder or bringing it's own malicious DLL.
+    This behavior may indicate an attempt to execute remotely hosted, potentially malicious files through deno.
+references:
+    - https://taggart-tech.com/evildeno/
+author: Josh Nickels, Michael Taggart
+date: 2025-05-22
+tags:
+    - attack.execution
+    - attack.t1204
+    - attack.t1059.007
+    - attack.command-and-control
+    - attack.t1105
+logsource:
+    category: file_event
+    product: windows
+detection:
+    selection_path_https:
+        TargetFilename|contains: '\deno\remote\https'
+    selection_path_dll:
+        TargetFilename|contains: '\deno\gen\'
+    selection_appdata:
+        TargetFilename|contains: '\appdata\'
+    condition: 1 of selection_path_* and selection_appdata
```

```suggestion
    selection_path:
        TargetFilename|contains:
            - '\deno\gen\'
            - '\deno\remote\https\'
        TargetFilename|contains|all:
            - ':\Users\
            - '\AppData\'
    condition: selection_path
```
you can just use a list. And as the AppData path also uses the `|all` modifier that makes the "or" list to an "and" list you can even use the same field twice in the same selection.